.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```diff
@@ -65,6 +65,9 @@ steps:
       - label: ":windows: x-pack/winlogbeat Win 2019 Unit Tests"
         key: "mandatory-win-2019-unit-tests"
         command: |
+          Push-Location -Path .buildkite/scripts
```

Comment:
Is there a reason why we change the dir to `.buildkite/scripts`? 
Can we execute `.buildkite/scripts/install_sysmon.ps1`?